tests/components/unifi/test_switch.py

```diff
@@ -317,8 +321,8 @@
     await hass.services.async_call(
         "switch", "turn_on", {"entity_id": "switch.block_client_1"}, blocking=True
     )
-    assert len(controller.mock_requests) == 11
-    assert controller.mock_requests[7] == {
+    assert len(controller.mock_requests) == 5
+    assert controller.mock_requests[4] == {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4925854)